packaging/os/dnf.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/python -tt
-# -*- coding: utf-8 -*-
-
-# Written by Cristian van Ee <cristian at cvee.org>
+# vim: set fileencoding=utf-8
+#
```

Comment:
you removed -*- coding: utf-8 -*-,  please add back as that should be the 2nd line for python's use to set file encoding